### DISCUSSION TITLE
Does it support translating from English to a foreign language?

### DISCUSSION THREAD

**Discussion by aparatext** [openai/whisper#110] [Q&A] [answered]
I read the documentation, but it only mentions translating to English. Can it do it in reverse? This would be very useful in subtitle generation.

**Comment by mayeaux** [comment]
Doesn't seem it's possible, if you look at the diagram of the model

**Comment by jongwook** [comment, marked as answer]
The model is not trained for that; it occasionally does it when you set `task="translate"` and `language` as the target language, but it's mostly due to imperfect language id in the training data, and it won't be very reliable.
